R/utils-shared.R

```diff
@@ -175,10 +175,21 @@ formatComments <- function(comments){
 ############ used in various places ############ 
 
 isEmpty <- function(val){
-  if(!is.null(val) && !is.na(val)){
-    result <- FALSE
+  if(is.null(val) || is.na(val)){
```

Comment:
You can assign what's in the if clause directly to the result variable. You don't have to do an if/else if all it does is assign true/false.